packages/manager/modules/billing/src/autoRenew/actions/terminate-all-dom/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,11 @@
+{
+  "autorenew_all_dom_terminate_title": "Vous avez demandé la résiliation de votre pack alldom",
+  "autorenew_all_dom_terminate_info": "Attention, la résiliation de ce pack n'entraîne pas la résiliation des domaines qui y sont attachés.",
+  "autorenew_all_dom_terminate_choose_option": "Veuillez sélectionner les noms de domaines que vous souhaitez résilier ci-dessous :",
+  "autorenew_all_dom_terminate_select_all_dom": "All dom case",
```

Comment:
**questions:** Shouldn't be translated too?

Reply:
It is shown like this in mock-up and no translation was given. https://xd.adobe.com/view/f4fbaec7-d0fd-4c67-a5ef-300f79b73b7d-a1a8/screen/b887906a-de93-4815-bfd3-5878bbba953b/

I will check with ECOM team if translation is required.